build/git_revision_test.go

```diff
@@ -59,8 +69,12 @@ func TestGitRevision_terraform(t *testing.T) {
 	}
 
 	// check if license was copied
-	if _, err := os.Stat(licensePath); err != nil {
-		t.Fatalf("expected license file not found at %q: %s", licensePath, err)
+	files, err := os.ReadDir(tempDir)
+	if err != nil {
+		t.Fatal("failed to read license directory", err)
+	}
+	if len(files) == 0 {
+		t.Fatalf("expected license file not found, tempDir %q is empty", tempDir)
```

Comment:
Is there a particular reason behind this change?

Reply:
Addressed in comment above